src/udev/udevadm-test.c

```diff
@@ -127,8 +127,8 @@ int test_main(int argc, char *argv[], void *userdata) {
 
         event = udev_event_new(dev, 0, NULL);
 
-        sigfillset(&mask);
-        sigprocmask(SIG_SETMASK, &mask, &sigmask_orig);
+        assert_se(sigfillset(&mask) == 0);
+        assert_se(sigprocmask(SIG_SETMASK, &mask, &sigmask_orig) == 0);
```

Comment:
not that it matters, but I usually prefer checking with `< 0` for failure and `>= 0` for success... But I guess == 0 works too...

Reply:
`>= 0` is good for our internal functions. But the glibc functions are documented to only return 0 or -1, so I think `==0` is appropriate. (And other glibc functions return >0 on error, so it just seems better to use `==0`).

Reply:
well, I'd always use the same checks.. I mean the general rule applies that negative means failure, and positive and zero means success, both for our own calls and most of glibc's calls. for example open() works that way and so does rename() and also the signal mask operations... hnece, given that these apis are mostly systematic on that aspect I'd stick to that aspect...

(yes there are exceptions, for example the pthread apis do return failure as positive, but they are the outlier there)

Reply:
> sigemptyset(),  sigfillset(),  sigaddset(),  and sigdelset() return 0 on success and -1 on error.

If we know that it returns exactly 0, I don't see a reason to check for anything else.